### DISCUSSION TITLE
Configure theme on client-side

### DISCUSSION THREAD

**Discussion by sumanthratna** [upptime/upptime#298] [Ideas] [answered]
Hi, 

It would be nice if Upptime had a simple dropdown somewhere on the page that indicates what theme is being used, which would allow users to change the theme based on their own preferences. The initial theme (i.e., default theme) should be based on the user's preference using `prefers-color-scheme` in CSS or [an alternative for JS](https://www.freecodecamp.org/news/how-to-detect-a-users-preferred-color-scheme-in-javascript-ec8ee514f1ef/)

**Comment by titanism** [comment, marked as answer]
https://github.com/upptime/status-page/pull/517

**Reply by AnandChowdhary** [reply]
This should be landing soon!
